docs/guides.md

```diff
@@ -15,36 +15,36 @@ On this page, we want to describe how to write new guides.
 The Guide JSON files are add-on JSON objects, with type "guide" and a "guide"
```

Comment:
I am sorry for all of these Markdown formatting changes. VSCode just did that automatically on save (?). If it's a problem I can rever them.